checks/evaluation/ci_tests_test.go

```diff
@@ -16,441 +16,153 @@ package evaluation
 import (
 	"testing"
 
-	"github.com/ossf/scorecard/v4/checker"
-	"github.com/ossf/scorecard/v4/clients"
+	sce "github.com/ossf/scorecard/v4/errors"
+	"github.com/ossf/scorecard/v4/finding"
 	scut "github.com/ossf/scorecard/v4/utests"
 )
 
-func Test_isTest(t *testing.T) {
+// Tip: If you add new findings to this test, else
+// add a unit test to the probes with the same findings.
+func TestCITests(t *testing.T) {
 	t.Parallel()
-	type args struct {
-		s string
-	}
 	tests := []struct {
-		name string
-		args args
-		want bool
+		name     string
+		findings []finding.Finding
+		result   scut.TestReturn
 	}{
 		{
-			name: "appveyor",
-			args: args{
-				s: "appveyor",
-			},
-			want: true,
-		},
-		{
-			name: "circleci",
-			args: args{
-				s: "circleci",
-			},
-			want: true,
-		},
-		{
-			name: "jenkins",
-			args: args{
-				s: "jenkins",
-			},
-			want: true,
-		},
-		{
-			name: "e2e",
-			args: args{
-				s: "e2e",
-			},
-			want: true,
-		},
-		{
-			name: "github-actions",
-			args: args{
-				s: "github-actions",
-			},
-			want: true,
-		},
-		{
-			name: "mergeable",
-			args: args{
-				s: "mergeable",
-			},
-			want: true,
-		},
-		{
-			name: "packit-as-a-service",
-			args: args{
-				s: "packit-as-a-service",
-			},
-			want: true,
-		},
-		{
-			name: "semaphoreci",
-			args: args{
-				s: "semaphoreci",
-			},
-			want: true,
-		},
-		{
-			name: "test",
-			args: args{
-				s: "test",
-			},
-			want: true,
-		},
-		{
-			name: "travis-ci",
-			args: args{
-				s: "travis-ci",
-			},
-			want: true,
-		},
-		{
-			name: "azure-pipelines",
-			args: args{
-				s: "azure-pipelines",
+			name: "Has CI tests. 1 tested out of 1 merged",
+			findings: []finding.Finding{
+				{
+					Outcome:  finding.OutcomePositive,
+					Probe:    "testsRunInCI",
+					Message:  "CI test found: pr: 1, context: e2e",
+					Values:   map[string]int{"totalMerged": 1, "totalTested": 1},
+					Location: &finding.Location{Type: 4},
+				},
 			},
-			want: true,
-		},
-		{
-			name: "non-existing",
-			args: args{
-				s: "non-existing",
+			result: scut.TestReturn{
+				Score:        10,
+				NumberOfInfo: 1,
 			},
-			want: false,
 		},
-	}
-	for _, tt := range tests {
-		tt := tt
-		t.Run(tt.name, func(t *testing.T) {
-			t.Parallel()
-			if got := isTest(tt.args.s); got != tt.want {
-				t.Errorf("isTest() = %v, want %v for test %v", got, tt.want, tt.name)
-			}
-		})
-	}
-}
-
-func Test_prHasSuccessfulCheck(t *testing.T) {
-	t.Parallel()
-
-	tests := []struct {
-		name    string
-		args    checker.RevisionCIInfo
-		want    bool
-		wantErr bool
-	}{
 		{
-			name: "check run with conclusion success",
-			args: checker.RevisionCIInfo{
-				PullRequestNumber: 1,
-				HeadSHA:           "sha",
-				CheckRuns: []clients.CheckRun{
-					{
-						App:        clients.CheckRunApp{Slug: "test"},
-						Conclusion: "success",
-						URL:        "url",
-						Status:     "completed",
-					},
+			name: "Has CI tests. 3 tested out of 4 merged",
```

Comment:
this needs an outcome negative right?

Reply:
Added in https://github.com/ossf/scorecard/pull/3621/commits/37abb3ebf085b0eaae9e15d1711fba2c30cf4e92